src/views/VallisFish.vue

```diff
@@ -14,41 +14,43 @@
           <thead>
             <tr>
               <th title="The name of the fish">Fish Name</th>
-              <th title="The model of servofish">
-                Model
+              <th title="Most common model of servofish">
+                Basic
               </th>
-              <th title="Amount of scraps you will receive when dismantling, dependent on model">
-                Scrap
+              <th title="Slightly more uncommon model of servofish">
+                Adorned
               </th>
-              <th title="Solaris United standing gain when donating fish">Standing</th>
-              <th title="You will always receive 1 of these per fish when dismantling">
+              <th title="Perfect condition of servofish model">
+                Magnificent
+              </th>
+              <th title="You will always receive 1 of these per fish regardless of model when dismantling">
                 Unique
               </th>
-              <th title="Location of where to find the fish">Location</th>
-              <th title="Time of when you can find the fish">Time</th>
+              <th title="Location of where to find the servofish">Location</th>
+              <th title="Cycle of when you can find the servofish">Time</th>
               <th title="How likely the fish will spawn">Rarity</th>
+              <th title="What bait will make this servofish more likely to spawn">Bait</th>
               <th title="The maximum points possible for this fish">
                 Max Points
               </th>
             </tr>
           </thead>
           <tbody v-for="(fish, index) in this.fishes" :key="fish.name">
             <tr :class="'color' + ((index % 2) + 1)">
-              <td rowspan="3">
-                <b-btn :id="`${fish.name}_tooltip`" size="md" variant="link" class="m-3">
+              <td>
+                <b-btn :id="`${fish.name}_tooltip`" size="md" variant="link">
                   {{ fish.name }}
                 </b-btn>
                 <b-tooltip :target="`${fish.name}_tooltip`" placement="top">
                   <FishImg type="fish" :item="fish.thumb" :name="fish.name" width="200" v-if="fish.thumb" />
                   <span v-else> No image available</span>
                 </b-tooltip>
               </td>
-              <td v-if="fish.smallLabel">{{ fish.smallLabel }}</td>
-              <td v-else>Basic</td>
-              <td>{{ fish.small.resources.scrap }}</td>
-              <td>{{ fish.small.standing }}</td>
-              <td rowspan="3">
-                <b-btn :id="`${fish.unique.name}_unique_tooltip`" size="md" variant="link" class="m-3">
+              <td>{{ fish.small.resources.scrap }} Scrap + {{ fish.small.standing }} Standing</td>
```

Comment:
instead of writing out each bit, i believe there's a way to map data to template for b-table so we can use sorting still...
ah, here we go: [scoped templates](https://bootstrap-vue.js.org/docs/components/table/#scoped-field-slots)